modules/data.atmosphere/R/met.process.R

```diff
@@ -259,20 +259,24 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
                               slat=new.site$lat,slon=new.site$lon,newsite=new.site$id)
 
   }else if(register$scale=="site"){ ##### Site Level Processing
-
-    logger.info("Gapfilling") # Does NOT take place on browndog!
-
-    input.id   <- cf.id[1]
-    outfolder  <- file.path(dir,paste0(met,"_CF_gapfill_site_",str_ns))
-    pkg        <- "PEcAn.data.atmosphere"
-    fcn        <- "metgapfill"
-    formatname <- 'CF Meteorology'
-    mimetype   <- 'application/x-netcdf'
-    lst        <- site.lst(site,con)
-
-    ready.id   <- convert.input(input.id,outfolder,formatname,mimetype,site.id=site$id,start_date,end_date,pkg,fcn,
-                                username,con=con,hostname=host$name,browndog=NULL,write=TRUE,lst=lst)
-
+    if(!is.null(register$gapfill)){
```

Comment:
Explain what you mean by 'too specific to ameriflux'. The function works on CF standard met on the standard set of variables, so why should it matter where that product's coming from? What in the function is actually Ameriflux specific?

I'm definitely not asking you to go off on every tangent, but you do need to let people know when their code is causing problems, rather than just hacking around it. In this case I'm guessing that part of the problem is that the PalEON met doesn't need to be gapfilled and you'd like the subroutine to detect that and bail earlier? If so submit an issue about this and tag @ankurdesai.  If the metgapfill is actually crashing on files that have no gaps thats a different bug, but it likewise needs an issue so that @ankurdesai knows he has code to fix and preferably more information (error messages, details of the example that's causing the problem, etc).